lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -259,6 +259,7 @@ def self.append_v1trigger(definition, v1trigger)
       end
 
       # Values for all Trigger Types
+      iTrigger.RandomDelay         = "PT#{v1trigger['random_minutes_interval']}M" unless v1trigger['random_minutes_interval'].nil?   || v1trigger['random_minutes_interval'].zero?
```

Comment:
Could you fill in the docs on this commit as well.  There's nothing anywhere that mentions a user may set this.

Reply:
Prior to this being hooked up with the provider, they definitely cannot. Where would that doc update land in this workset? Documenting it in this helper seems like the wrong place but we've pushed off updating the type/provider in this ticket for the wire-up ticket.

Reply:
Yeah, that's a good point.  Of course, users won't really get their hands on the module until the code actually ships, at which point the code will be integrated.

I don't feel super strongly either way, but it's usually helpful to update docs when a code change lands that enables a feature?

Reply:
Addressed with a minor readme update: